DER/SecurityBasedDerivatives/SecurityBasedDerivatives.rdf

```diff
@@ -179,7 +131,7 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>equity derivative</rdfs:label>
-		<skos:definition>security-based derivative whose underlier is an individual share, an equity index, or a custom basket of equity assets</skos:definition>
+		<skos:definition>security-based derivative whose underlier is an individual share, equity index, or custom basket of equity assets</skos:definition>
```

Comment:
again should not enumerate subclasses

Reply:
Will work on a better definition in our next round